webapp/app/templates/basis/base_step_form.html

```diff
@@ -39,10 +39,4 @@
         $('input[type="date"]').attr('max', '9999-12-31');
     </script>
     <script src="{{ url_for('static', filename='js/jquery.mask.min.js') }}" type=text/javascript></script>
```

Comment:
Correct me if I'm wrong, but I think you need to convert the mask on `.date-input` to a `data-mask` as well if you delete it (which I think would be good).

Reply:
I think we do not need to do this because each subfield of the date is a NumericInput with the correct length and therefore only allows digits anyway.